system_tests/block_validator_test.go

```diff
@@ -113,6 +114,8 @@ func testBlockValidatorSimple(t *testing.T, dasModeString string, expensiveTx bo
 	if !nodeB.BlockValidator.WaitForBlock(ctx, lastBlock.NumberU64(), timeout) {
 		Fail(t, "did not validate all blocks")
 	}
+
+	lifecycleManager.StopAndWaitUntil(time.Second)
```

Comment:
Would be nice to shutdowns as defers to make sure we do cleanup and make things easier to track

Reply:
Fixed.